src/components/Success/Success.module.scss

```diff
@@ -0,0 +1,105 @@
+.successComponent{
+    background: #000000;
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    .avatarProfile{
+        background: url('./Particles/avatar-profile.svg');
+        position: absolute;
+        width: 40px;
+        height: 40px;
+        right: 0px;
+        top: 0px;
+    }
+
+    .rectangle{
+        display:flex;
+        justify-content: center;
+        align-content: center;
+        position:absolute;
+        width: 450px;
+        height:600px;
+        background: linear-gradient(180deg, #29AC6D 0%, rgba(41, 172, 109, 0.47) 100%);
+        border-radius: 4px;
+        
+
+        .guitarist{
+            background:url('./Particles/electric-guitar.svg');
+            position: absolute;
+            width: 200px;
+            height: 200px;
+            left: 125px;
+            top: 200px;
+        }
+
+        .successText{
+            background: url('./Particles/SuccessText.svg');
+            position: absolute;
+            width: 136px;
+            height:23.4px;
+            left: 160px;
+            top: 450px;
+            
+            font-family: Biryani;
+            font-style: normal;

```

Comment:
font-style and font-weight are normal by default. So you need to remove it. Also, font-family is already added to global CSS, so you can remove it here.